api/DataAccess/DataAccessObjects/UserDataAccessObject.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Data.Context;
+using Data.Entities;
+using DataAccess.Interfaces;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DataAccess.DataAccessObjects
+{
+    public class UserDataAccessObject : IUserDataAccessObject
+    {
+        protected readonly UnoOnBoardingContext _context;
+
+        public UserDataAccessObject(UnoOnBoardingContext context)
+        {
+            _context = context;
+        }
+
+        public async Task<List<User>> FilterUsers(string search,int sort)
+        {
+            if (sort == 0)
+            {
+                var result = await _context.Set<User>()
+                .Where(x => (x.Name!.ToLower().Contains(search.ToLower())) ||
+                            (x.Email!.ToLower().Contains(search.ToLower())))
+                .OrderBy(x => x.Name).ToListAsync();
+
+                return result.Where(x => !x.IsDeleted).ToList(); ;
+            }
+
+            else
+            {
+                var result = await _context.Set<User>()
+                .Where(x => (x.Name!.ToLower().Contains(search.ToLower())) ||
```

Comment:
pay attention to words with accents: "Antônio" does not contain "Antonio"

Reply:
added extension method